app/views/article/_index_default.html.erb

```diff
@@ -22,7 +22,14 @@
 <% if document.access_panels.online? && !document.eds_restricted? %>
   <ul class='document-metadata dl-horizontal dl-invert'>
     <% document.access_panels.online.links.each do |link| %>
-      <li><%= link_to(link.text, link.href) %></li>
+      <li>
+        <% if link.text =~ /^Find it in print/ -%>
```

Comment:
Also, in my WIP PR, I refactored the EdsLinks module and it'd be easier to add it with that code there, so maybe we should review it first -- #1537